components/reaction-badge.json

```diff
@@ -0,0 +1,20 @@
+{
+  "reaction-badge": {
```

Comment:
One final request: can you change this to either:
{
  "reactionbadge": {
  ...
  }
}
or
{
  "reaction": {
    "badge": {
     ...
    }
  }
}
depending on whether you think there will be more reaction tokens. Do you know if the buttons on the reaction popup are just standard buttons, or will we need to define them as a special case?

Reply:
Yeah, in which case I think you should do
{
  "reaction": {
    "badge": {
     ..
    },
    "button": {
     ...
   }
}